src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/SystemProgressCasesToAosOverdueTask.java

```diff
@@ -49,7 +49,8 @@ public void run() {
         final String serviceAuth = authTokenGenerator.generate();
 
         try {
-            final List<CaseDetails> casesInAwaitingAosState = ccdSearchService.searchForAllCasesWithStateOf(AwaitingAos, user, serviceAuth);
+            final List<CaseDetails> casesInAwaitingAosState =
+                ccdSearchService.searchForAllCasesWithStateOf(AwaitingAos, null, user, serviceAuth);
```

Comment:
Maybe there should be two search methods, one that has the flag parameter and one without.  That way we don't have to pass the null as an argument, internally the methods can call a single implementation.